astropy/table/tests/test_showtable.py

```diff
@@ -28,11 +29,20 @@ def test_info(capsys):
 def test_stats(capsys):
     showtable.main([os.path.join(FITS_ROOT, 'data/table.fits'), '--stats'])
     out, err = capsys.readouterr()
-    assert out == ('<Table length=3>{0}'
-                   ' name    mean    std   min  max {0}'
-                   '------ ------- ------- ---- ----{0}'
-                   'target      --      --   --   --{0}'
-                   ' V_mag 12.8667 1.72111 11.1 15.2{0}').format(os.linesep)
+    if NUMPY_LT_1_14:
```

Comment:
We really shouldn't be testing float output!

Reply:
We can maybe use another table, could you check the output with `astropy/io/ascii/tests/t/simple_csv.csv` ? (I don't have time to install numpy 1.14 right now, maybe later today or tomorrow):
```
❯ showtable astropy/io/ascii/tests/t/simple_csv.csv --stats
<Table length=2>
name mean std min max
---- ---- --- --- ---
   a  2.5 1.5   1   4
   b  3.5 1.5   2   5
   c  4.5 1.5   3   6
```

Reply:
I think this is a case where we *should* be testing non-nice floating point values.  The whole point of pprint and showtable is creating human-readable output that is informative and "pleasant".  So using real-world values is important to catch formatting issues that our users will end up seeing.

Reply:
@taldcroft, @saimn - yes, although it may be that we have to define what is pleasant ourselves rather than rely on numpy doing it for us. But let's discuss in #6962, as it is somewhat orthogonal to the point of this PR, which is simply to get things to pass with numpy 1.14.x.

Reply:
Agreed about moving the discussion to #6962 , so the current change is fine by me.